private/bufpkg/bufimage/bufimagemodify/php_namespace_test.go

```diff
@@ -49,14 +49,50 @@ func TestPhpNamespaceEmptyOptions(t *testing.T) {
 		assertFileOptionSourceCodeInfoEmpty(t, image, phpNamespacePath, false)
 
 		sweeper := NewFileOptionSweeper()
-		modifier := PhpNamespace(sweeper)
+		modifier := PhpNamespace(sweeper, nil)
 		err := modifier.Modify(
 			context.Background(),
 			image,
 		)
 		require.NoError(t, err)
 		assert.Equal(t, testGetImage(t, dirPath, false), image)
 	})
+
+	t.Run("with SourceCodeInfo and per-file overrides", func(t *testing.T) {
+		t.Parallel()
+		image := testGetImage(t, dirPath, true)
+		assertFileOptionSourceCodeInfoEmpty(t, image, phpNamespacePath, true)
+
+		sweeper := NewFileOptionSweeper()
+		phpNamespaceModifier := PhpNamespace(sweeper, map[string]string{"a.proto": "override"})
```

Comment:
Given that we have `a.proto` in _a lot_ of the tests, let's make sure that we're actually testing whether or not this is _only_ affecting `a.proto` and not others. In this test, we're using the `emptyoptions` directory which only contains a single `a.proto`, so we don't actually know whether or not this file is being overridden while others aren't.

Adding another file in each of these relevant `testdata` directories and asserting that it doesn't adopt the `override` value is sufficient.